can-view-parser.js

```diff
@@ -347,7 +347,15 @@ var callAttrStart = function(state, curIndex, handler, rest, lineNo){
 
 var callAttrEnd = function(state, curIndex, handler, rest, lineNo){
 	if(state.valueStart !== undefined && state.valueStart < curIndex) {
-		handler.attrValue(rest.substring(state.valueStart, curIndex), lineNo);
+		var val = rest.substring(state.valueStart, curIndex);
+		//!steal-remove-start
+		var quotedVal = rest.substring(state.valueStart - 1, curIndex + 2);
+		quotedVal = quotedVal.trim();
+		if (state.inQuote !== quotedVal.charAt(quotedVal.length - 1)) {
+			dev.warn("End quote is missing for " + val);
```

Comment:
Should we add the filename and line number, like the other errors?

Reply:
Filename and line number of the file caused the warning?

Reply:
Sorry. Kevin asked me to review this, so I assumed it was his code. We've been trying to make debugging easier, and among that is to make errors more explanatory. As of today, these errors can have the [filename and line number](https://github.com/canjs/can-view-parser/blob/3b839f00567d174e1d8cf590834d41953e5be9d5/can-view-parser.js#L155-L160) (though the filename is not always present).

Reply:
No problem, for the line I take ```lineNo``` argument as it is or it should be processed?

Reply:
Shouldn't have to do anything with it. Should just be able to add it on like in the example.